src/strategies/ens-all-club-digits/examples.json

```
@@ -0,0 +1,20 @@
+[
+  {
+    "name": "ENS ALL Club Digits",
+    "strategy": {
+      "name": "ens-all-club-digits",
+      "params": {
+        "address": "0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147eA85",
+        "symbol": "NUMS",
+        "numberOfDigits": 4
```

changing this to test
```suggestion
        "numberOfDigits": 7
```